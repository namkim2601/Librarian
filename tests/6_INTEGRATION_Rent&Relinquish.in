ReNt nomembers nobooks
ReLiNqUiSh nomembers nobooks
add member member1
rent memberexists nobooks
relinquish memberexists nobooks
add collection collection.csv
rent nosuchmember 111111
relinquish nosuchmember 111111
rent 100000 nosuchbook
relinquish 100000 nosuchbook
relinquish 100000 111111
rent 100000 111111
rent 100000 111111
relinquish 100000 111111
exit


