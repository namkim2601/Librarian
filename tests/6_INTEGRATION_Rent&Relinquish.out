user: No members in system.

user: No members in system.

user: Success.

user: No books in system.

user: No books in system.

user: 17 books successfully added.

user: No such member in system.

user: No such member in system.

user: No such book in system.

user: No such book in system.

user: Unable to return book.

user: Success.

user: Book is currently unavailable.

user: Success.

user: Ending Library process.
